src/particle_header.F90

```diff
@@ -90,10 +94,10 @@ module particle_header
     type(Bank) :: secondary_bank(MAX_SECONDARY)
 
   contains
-    procedure :: initialize => initialize_particle
-    procedure :: clear => clear_particle
-    procedure :: initialize_from_source
-    procedure :: create_secondary
+    procedure, pass :: initialize => initialize_particle
+    procedure, pass :: clear => clear_particle
+    procedure, pass :: initialize_from_source => initialize_from_source
+    procedure, pass :: create_secondary => create_secondary
```

Comment:
Eh, I was used to doing it per a previous style guide I worked under, and in general I lean towards more explicit over less explicit (another example of this are my clear routines)